src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,27 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
```

Comment:
The `file` variable is initialized with `fromFileName`, which is correct for reading. However, ensure you use `toFileName` for writing the output.